src/vs/workbench/contrib/mergeEditor/test/browser/projection.test.ts

```diff
@@ -38,6 +38,7 @@ this.beakContainer.className = 'status-bar-beak-container';
 2
 3 // Beak Container
 4 this.beakContainer = document.createElement('div');
+䷅
```

Comment:
Is that an intentional weird character change?

Reply:
Yes, see here: https://github.com/microsoft/vscode/pull/158283/files#diff-b85b02f00694da1c9fd7d4f71eed3156dd13189c0fdbd83771deea4f63654aacR130 .